pkg/controller/node/status.go

```diff
@@ -73,6 +85,20 @@ func calculateStatus(pool *mcfgv1.MachineConfigPool, nodes []*corev1.Node) mcfgv
 		supdating := mcfgv1.NewMachineConfigPoolCondition(mcfgv1.MachineConfigPoolUpdating, corev1.ConditionTrue, fmt.Sprintf("All nodes are updating to %s", pool.Status.Configuration.Name), "")
 		mcfgv1.SetMachineConfigPoolCondition(&status, *supdating)
 	}
+
+	if len(degradedMachines) > 0 {
+		sdegraded := mcfgv1.NewMachineConfigPoolCondition(mcfgv1.MachineConfigPoolDegraded, corev1.ConditionTrue, fmt.Sprintf("%d nodes are reporting degraded status on sync", len(degradedMachines)), strings.Join(degradedReasons, ","))
```

Comment:
what does sdegraded mean?

Reply:
maybe rename?

Reply:
status degraded, same as supdated, supdating above - just being consistent

Reply:
> maybe rename?

it looks sane and consistent with the others in this context, we're switching status here to (updating,updated,degraded). Longer would be `statusDegraded` or something like this. I'm ok with that but let's follow up as other needs changes as well (above) and doesn't belong to this PR

Reply:
yeah i had no clue that was supposed to mean status degraded. sounds good